src/main/java/frc/robot/RobotContainer.java

```diff
@@ -280,17 +306,16 @@ private void configureBindings() {
             masher.povUp();
         }
         if (FeatureFlags.runDrive) {
-            masher.povUp()
-                    .onTrue(new InstantCommand(() -> drive.setAlignTarget(AlignTarget.SPEAKER)));
+            masher.povUp().onTrue(new InstantCommand(() -> drive.setAlignTarget(AlignTarget.UP)));
 
             masher.povRight()
-                    .onTrue(new InstantCommand(() -> drive.setAlignTarget(AlignTarget.AMP)));
+                    .onTrue(new InstantCommand(() -> drive.setAlignTarget(AlignTarget.RIGHT)));
 
             masher.povLeft()
-                    .onTrue(new InstantCommand(() -> drive.setAlignTarget(AlignTarget.SOURCE)));
+                    .onTrue(new InstantCommand(() -> drive.setAlignTarget(AlignTarget.LEFT)));
 
             masher.povDown()
-                    .onTrue(new InstantCommand(() -> drive.setAlignTarget(AlignTarget.ENDGAME)));
+                    .onTrue(new InstantCommand(() -> drive.setAlignTarget(AlignTarget.DOWN)));
```

Comment:
Up/Down/Left/Right aren't intuitive alignment targets like speaker/amp/source/endgame. Could we go back to using the intuitive game targets rather than the POV directions?

Reply:
switched up(speaker) right(amp), left(source), down(endgame), and added right joystick trigger as aligning true